src/index.js

```diff
@@ -12,7 +12,7 @@ class exampleBot extends SapphireClient {
       partials: ["CHANNEL"],
     });
 
-    container.slashCommands = new SlashCommandStore();
+    this.stores.register(new SlashCommandStore());
```

Comment:
Intended way like in @sapphire/framework, see;
https://github.com/sapphiredev/framework/blob/main/src/lib/SapphireClient.ts#L235

Not intended to be done in the super of a Store.